### PR TITLE
feat: add navigation component

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Navigation Component**
> Responsive Navigation Bar

navigation

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -13,3 +13,4 @@
 @import url("../components/input/input.css");
 @import url("../components/text-utilities/text-util.css");
 @import url("../components/lists/lists.css");
+@import url("../components/navigation/navigation.html");
```
Comment (dev-enforced) [inline styles/style.css]: Here you shall mention navigation.css instead of .html

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+.kal_nav_list {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.kal_nav_item {
+    padding-right: var(--sp-0-75);
+}
+
+.kal_nav_item a {
```
Comment (dev-enforced) [inline components/navigation/navigation.css]: You can rename this class as .kal_nav_item_link {
style rules.
} and avoid using a for alloting the styles to it

File: components/navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    color: var(--default-color);
+}
+.kal_nav_item a:hover {
```
Comment (dev-enforced) [inline components/navigation/navigation.css]: Same as above

**Review by dev-enforced — APPROVED** [review]
Looks really good the navigation as shown from pictures
The code also looks well structured along with media queries for responsive nature and is good to merge.
Suggested some minor changes

File: components/navigation/navigation.css (hunk 2 above)
Comment (nerdyspook) [reply components/navigation/navigation.css]: I have declared a class for them but forgot to change it in CSS, thanks again.

File: styles/style.css (hunk 1 above)
Comment (nerdyspook) [reply styles/style.css]: Thanks for noticing that.

**Review by ajay-mandaviya — APPROVED** [review]

**Comment by ajay-mandaviya** [comment]
all good merge it

**Review by Hardiegogo — APPROVED** [review]
Looks clean!

**Pull request merged**